src/pages/iou/request/MoneeRequestParticipantsSelector.js

```diff
@@ -0,0 +1,315 @@
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import lodashGet from 'lodash/get';
+import ONYXKEYS from '../../../ONYXKEYS';
+import styles from '../../../styles/styles';
+import OptionsSelector from '../../../components/OptionsSelector';
+import * as OptionsListUtils from '../../../libs/OptionsListUtils';
+import * as ReportUtils from '../../../libs/ReportUtils';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import * as Browser from '../../../libs/Browser';
+import compose from '../../../libs/compose';
+import CONST from '../../../CONST';
+import personalDetailsPropType from '../../personalDetailsPropType';
+import reportPropTypes from '../../reportPropTypes';
+import refPropTypes from '../../../components/refPropTypes';
+
+const propTypes = {
+    /** Beta features list */
+    betas: PropTypes.arrayOf(PropTypes.string),
+
+    /** Callback to request parent modal to go to next step, which should be split */
+    onFinish: PropTypes.func.isRequired,
+
+    /** A ref to forward to options selector's text input */
+    forwardedRef: refPropTypes,
+
+    /** Callback to add participants in MoneyRequestModal */
+    onParticipantsAdded: PropTypes.func.isRequired,
+
+    /** Selected participants from MoneyRequestModal with login */
+    participants: PropTypes.arrayOf(
+        PropTypes.shape({
+            accountID: PropTypes.number,
+            login: PropTypes.string,
+            isPolicyExpenseChat: PropTypes.bool,
+            isOwnPolicyExpenseChat: PropTypes.bool,
+            selected: PropTypes.bool,
+        }),
+    ),
+
+    /** All of the personal details for everyone */
+    personalDetails: PropTypes.objectOf(personalDetailsPropType),
+
+    /** All reports shared with the user */
+    reports: PropTypes.objectOf(reportPropTypes),
+
+    /** padding bottom style of safe area */
+    safeAreaPaddingBottomStyle: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
+
+    /** The type of IOU report, i.e. bill, request, send */
+    iouType: PropTypes.oneOf(_.values(CONST.IOU.TYPE)).isRequired,
+
+    /** The request type, ie. manual, scan, distance */
+    iouRequestType: PropTypes.oneOf(_.values(CONST.IOU.REQUEST_TYPE)).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    participants: [],
+    forwardedRef: undefined,
+    safeAreaPaddingBottomStyle: {},
+    personalDetails: {},
+    reports: {},
+    betas: [],
+};
+
+function MoneeRequestParticipantsSelector({
```

Comment:
```suggestion
function MoneyRequestParticipantsSelector({
```

Reply:
This was done intentionally as part of my rollout strategy to reduce conflicts. I have [a phase 2 GH](https://github.com/Expensify/App/issues/29107) that will address things like this and remove the old components.

Reply:
@tgolen I know that the typo is intentional, but I feel like it's too close to the real word and I'm worried that between this and phase 2 going out quite a few of these will be renamed. Can we use something more obvious so others don't revert this name ahead of time?

Reply:
Yes, we can use anything. What do you suggest?

Reply:
Maybe we can prefix the components name with 'Temporary'?

Reply:
That would work for me. Any last regrets before I rename everything?